configuration/observability.mdx

```diff
@@ -81,3 +81,37 @@ Currently, we support the following tracing backends:
 Enable tracing via the values described in the [Tracing configuration](/configuration/overview#tracing) and point Flipt to your configured collector to record spans.
 
 There are [examples](https://github.com/flipt-io/flipt/tree/main/examples/tracing) provided in the main GitHub repository showing how to set up Flipt with each of the supported tracing backends.
+
+## Audit Events
+
+Starting from `v1.21.0`, Flipt supports sending audit events to configured sinks. Audit events have the following structure:
+
+```json
+{
```

Comment:
i think this JSON needs updating now that we've changed the structure/fields a bit

Reply:
@markphelps Definitely. I'll do that.